src/bindings/java/com/zerotier/sockets/ZeroTierDatagramSocket.java

```diff
@@ -165,7 +165,7 @@ public void receive(DatagramPacket packet) throws IOException
             0,
             packet.getLength());
         if ((bytesRead <= 0) | (bytesRead == -104) /* EINTR, from SO_RCVTIMEO */) {
-            throw new IOException("read​(DatagramPacket), errno=" + bytesRead);
+            throw new IOException("read(DatagramPacket), errno=" + bytesRead);
```

Comment:
It may be hard to see the change here. A git client like Sublime Merge can display the stray U+200b characters

<img width="843" alt="Screenshot 2022-12-12 at 3 00 22 PM" src="https://user-images.githubusercontent.com/183619/207142745-e3e28d77-ad63-433b-8290-333645d0ff78.png">